reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -7899,6 +7899,51 @@ public final void subscribe(Subscriber<? super T> actual) {
 		return subscriber;
 	}
 
+	/**
+	 * Provide current {@link Flux<T>} transformation when the first available signal
+     * appears in the upstream. To make this transformation possible, the switch on the
+     * first operator proactively requests the first element from the upstream. It does
+     * not mean that upstream is always mandated to return an element, but free to be
+     * empty or error stream, so the given in transform bi-function {@link Signal} can be
+     * either onNext, onError or onComplete.
+     *
+     * Note that any given signal will be propagated to the downstream regardless the
+     * type of the signal. Typically, this operator could be used to provide stream
+     * transformation depends on the first signal in the stream.
+     *
+     * For example, it could be used to define the stream routing, depends on the first
```

Comment:
Replace the two line paragraph with the following:

```
     * For example, this operator could be used to define a form of routing, depending on the first
     * element (which could contain routing metadata for instance):
```

Reply:
done